woocommerce/google-listings-and-ads
# Upgrade the workflows in GitHub Actions to use Node.js v20

### Changes proposed in this Pull Request:

Closes #2218 

This PR upgrades the workflows in GitHub Actions to use Node.js v20.

### Detailed test instructions:

1. View the [Checks](https://github.com/woocommerce/google-listings-and-ads/pull/2404/checks) tab to check if all checks have passed and if there are no longer deprecated warnings about Node.js <= 16
2. View the [E2E Tests result](https://github.com/woocommerce/google-listings-and-ads/actions/runs/9110477176) that is not listed in the Checks tab
3. The rest of workflows that can't be viewed in this PR have basically been verified during development, if one wants to view them, please find the corresponding development RP in https://github.com/woocommerce/grow/issues/108

### Changelog entry
